auth.go

```diff
@@ -257,6 +257,26 @@ func (s *Service) AddDevProvider(port int) {
 	s.providers = append(s.providers, provider.NewService(provider.NewDev(p)))
 }
 
+// AddAppleProvider allow SignIn with Apple ID
+func (s *Service) AddAppleProvider(name string, appleConfig provider.AppleConfig, privKeyLoader provider.PrivateKeyLoaderInterface) error {
```

Comment:
Another provider creator is generic and requires a name because of that, I don't think this one needs it. @umputun ?

Reply:
yeat, bothered me as well. But i don't think it can be done with the same params as others

Reply:
I think it would be better to hardcode name=“apple” for consistency and not allow user changing it.

Reply:
agree

Reply:
What is about if a user want add more then one AppleProvider or that case not acceptable? I look to the `TelegramProvider` where a user can define a provider name.
If you think use an AppleProvider without name define by a user,  I will remove name define from AppleProvider constructor.

Reply:
> more then one AppleProvider

this would be ... unusual. Personally, I have never seen an appincation providing multiple google logins or multiple Facebook logins at the same time. I don't really see in what situation such functionality would be desirable. 

Regarding TelegramProvider - yeah, we should address it for the same reason and hardcode provider's name. I guess I missed this one during review because TG doesn't have a full integration like yours and wired in via a custom provider.

Reply:
Ok.  I will make hardcoded name for AppleProvider.